assets/components/aws-ebs-csi-driver/manifests/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
The `manifests` directory should be skipped: e.g. `assets/components/aws-ebs-csi-driver/.helmignore`.